packages/docs-reanimated/docs/guides/web-support.mdx

```diff
@@ -39,10 +62,10 @@ yarn web
 
 If you want to start the example applications from the
 [reanimated repository](https://github.com/software-mansion/react-native-reanimated)
-you need to run the following command inside the `Example` directory:
+you need to run the following command inside the `apps/web-example` directory:
 
 ```shell
-yarn start-web
+yarn start
```

Comment:
Haven't we skipped the `yarn && yarn build` in the repository root?

Reply:
i thought it's such a basic that everyone knows that, but sure, i'll add this!